odoo/addons/base/models/ir_actions.py

```diff
@@ -564,6 +566,12 @@ def run(self):
         """
         res = False
         for action in self:
+            action_groups = action.groups_id
+            if action_groups:
+                user_groups = self.env.user.groups_id
+                if not action_groups & user_groups:
+                    raise AccessError(_("You don't have access rights to run this action, Please contact administrator for more information."))
```

Comment:
@pka-odoo i would prefer "Not enough access rights to run this action.", then it's fine by me and we'll wait for raphael's next review